services-custom/s3-transfer-manager/src/test/java/software/amazon/awssdk/transfer/s3/internal/S3NativeClientConfigurationTest.java

```diff
@@ -37,15 +38,25 @@ public static void setup() {
         System.setProperty("aws.crt.debugnative", "true");
     }
 
+    // TODO: verify CRT resources are closed appropriately
+
+    @Test
+    public void defaultConfiguration_close_shouldShutdownDefaultExecutor() {
+        S3NativeClientConfiguration configuration = S3NativeClientConfiguration.builder().build();
+        configuration.close();
+        assertThat(configuration.futureCompletionExecutor()).isInstanceOf(ExecutorService.class);
+        assertThat(((ExecutorService) configuration.futureCompletionExecutor()).isShutdown()).isTrue();
```

Comment:
Nit: Perhaps overkill, but may want to also assert `isShutDown() == false` before closing.

Reply:
Yeah, sure